src/concat_text/script.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I prefer this:
```suggestion
#!/usr/bin/env bash
```
But not sure if it works on alpine.

Reply:
Good point, it was there but I changed (and forgot to revert it) while troubleshooting a bug.